src/common/schemas/types.ts

```diff
@@ -0,0 +1,13 @@
+enum FieldType {
+  DROPDOWN = 'dropdown',
+  INPUT = 'input',
+}
```

Comment:
Once you build the views as components, not purely driven from schemas... Remember to delete this nonsense ;)

Reply:
Lol yes...for now i've left in this kinda stuff as reference, and I'll 🔥 it once I get the components done 🙂